docs/dsl/avatar.rst

```diff
@@ -0,0 +1,113 @@
+avatar
+======
+
+Renders an avatar SVG image using using https://avatars.dicebear.com/. Uses the SVG-specified units and DPI to determine the pixel width and height.  If neither data nor file are specified for a given card, this method does nothing.
+
```

Comment:
Should also probably say that beyond the first few options, it's identical to `svg`.